ember-scoped-css/src/lib/scoped-css-preprocessor.js

```diff
@@ -82,15 +92,26 @@ class ScopedFilter extends Filter {
     for (let inputPath of this.inputPaths) {
       const templateFile = {};
 
-      for (let ext of TEMPLATE_EXTENSIONS) {
+      eachExtension: for (let ext of TEMPLATE_EXTENSIONS) {
         const templatePath = relativePath.replace(/\.css/, '.' + ext);
-        const templateFilePath = path.join(inputPath, templatePath);
-        const exists = await fsExists(templateFilePath);
+        let templateFilePath = path.join(inputPath, templatePath);
+        let exists = await fsExists(templateFilePath);
+
+        /**
+         * Pods support
+         */
+        if (ext === 'hbs' && !exists && relativePath.endsWith('styles.css')) {
+          let podsName = relativePath.replace(/styles\.css$/, 'template.hbs');
+
+          templateFilePath = path.join(inputPath, podsName);
+          exists = await fsExists(templateFilePath);
+        }
 
         if (exists) {
           templateFile.path = templateFilePath;
           templateFile.ext = ext;
-          continue;
```

Comment:
continue here, I believe was a mistake. once we find a file, we don't want to keep searching for more files.